vmm/src/seccomp_filters.rs

```diff
@@ -809,6 +809,7 @@ fn vcpu_thread_rules(
         (libc::SYS_rt_sigreturn, vec![]),
         (libc::SYS_sched_yield, vec![]),
         (libc::SYS_sendmsg, vec![]),
+        (libc::SYS_sendto, vec![]),
```

Comment:
Since adding this has security implications we probably want some way to opt out of introducing this rule. 

I imagine the "correct" way to do that would be to let `get_seccomp_filter` also take a callback 

```rust
impl FnMut(c_long) -> bool
```
 to opt out of certain things from the filter that you don't want during construction. That would require a separate PR though, so you can probably leave that out here for now, but please add a "TODO" comment about it.

Reply:
While I totally see the advantage of a "dynamic seccomp ruleset", this is totally out of scope of our PoC. This would require major refactorings in the whole CHV code base. The current model in CHV is to add all syscalls that are possible in theory.

When we start changing this for one single code path, we make things worse and not better, IMHO.